diff --git a/node_modules/sheryjs/src/Utils.js b/node_modules/sheryjs/src/Utils.js
index 5fffdff..f8a6685 100644
--- a/node_modules/sheryjs/src/Utils.js
+++ b/node_modules/sheryjs/src/Utils.js
@@ -1,5 +1,5 @@
 import * as THREE from "three"
-import ControlKit from "controlkit"
+import * as ControlKit from "controlkit"
 import { ScrollPos } from "./Effects"
 import gsap from "gsap"
 import { Expo } from "gsap/all"
